ngraph/frontend/paddlepaddle/src/op/prior_box.cpp

```diff
@@ -0,0 +1,111 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "ngraph/op/prior_box.hpp"
+
+#include <ngraph/opsets/opset8.hpp>
```

Comment:
To align opset version, better follow this way for future op conversion -
https://github.com/openvinotoolkit/openvino/blob/b968c7b813a2e89ccd944f76d02b0ddd709fbc02/ngraph/frontend/paddlepaddle/src/op/gelu.cpp#L7

Reply:
modified